.travis.yml

```diff
@@ -72,11 +72,12 @@ jobs:
 
 before_install:
   - export GALAXY_REPO=https://github.com/galaxyproject/galaxy
-  - export GALAXY_RELEASE=release_18.05
+  - export GALAXY_RELEASE=release_19.05
```

Comment:
20.05 is the latest release and python3

Reply:
@bgruening Thank you for the suggestion. I was trying different versions and all were failing except for `20.05` as it is for python version 3. Versions `18.05` and `19.05` failed at gcc linking steps. Hopefully this updated testing environment will last for some time .